docs/_templates/navigation.html

```diff
@@ -0,0 +1,9 @@
+{{ toctree(maxdepth=2, includehidden=theme_sidebar_includehidden, collapse=theme_sidebar_collapse) }}
```

Comment:
somehow these changes wound up in here?

Reply:
Somehow I reverted them, but they now were conflicting, so I had to revert the revert, now it looks clean. 💀